### ISSUE TITLE
How to print out raw numbers?

### ISSUE THREAD

**Issue by bluebat** [octaspire/dern#1]
FYI, the RPM package for linux: https://github.com/rpmsphere/x86_64/raw/master/d/dern-0.489.6-1.x86_64.rpm
BTW, how to print out raw numbers? i.e.
> (print {D+123})
123

**Comment by octaspire** [comment]
Sorry for the long delay. 

I implemented a way to print and format values (including numbers, characters, and strings) readably and in "raw" mode. Readable mode is the default (it doesn't mean human-readable, but instead that the printed value can be read back and evaluated in Dern). This is somewhat similar to Common Lisp's special variable `*print-readably*` that I used as an inspiration for naming the new Dern function `print-readably`. 

Change is implemented in Dern version 0.490.0 in commit 02d949b6db649abdc2bb021859aab57592e5b708, and isn't thus unfortunately available in the RPM package you mentioned (I don't maintain that package). So, at the moment the feature is available only when building from source, until the package is updated. 

Example usage:
```
(print-readably false)
(println {D+123})
(print-readably true)
(println {D+123})
```
